scripts/malleus.sh

```diff
@@ -135,6 +135,7 @@ case $1 in
         --switch-speakers) if [ -n "$optvalue" ]; then MALLEUS_SWITCH_SPEAKERS=$optvalue; else MALLEUS_SWITCH_SPEAKERS=true; fi;;
         --use-stage-view) if [ -n "$optvalue" ]; then MALLEUS_USE_STAGE_VIEW=$optvalue; else MALLEUS_USE_STAGE_VIEW=true; fi;;
         --headless) if [ -n "$optvalue" ]; then MALLEUS_USE_HEADLESS=$optvalue; else MALLEUS_USE_HEADLESS=true; fi;;
+        --save-logs) if [ -n "$optvalue" ]; then MALLEUS_SET_SAVELOGS=$optvalue; else MALLEUS_SET_SAVELOGS=false; fi;;
```

Comment:
This should be MALLEUS_SET_SAVELOGS=true so just passing `--save-logs` to the script causes it to save logs.

You'll also need to add MALLEUS_SET_SAVELOGS to the `set_defaults` function.